src/nanoemoji/write_font.py

```diff
@@ -269,12 +269,28 @@ def _update_paint_glyph(paint):
             return paint
 
         assert paint.glyph.startswith("M"), f"{paint.glyph} doesn't look like a path"
+        path_in_svg_space = paint.glyph
         path_in_font_space = (
-            SVGPath(d=paint.glyph).apply_transform(svg_units_to_font_units).d
+            SVGPath(d=path_in_svg_space).apply_transform(svg_units_to_font_units).d
         )
 
-        reuse_result = glyph_cache.try_reuse(path_in_font_space)
+        print("REUSE IN SVG SPACE")
+        print("  ", path_in_svg_space)
+        print("  ", path_in_font_space)
+        # The odds of finding reuse in svg space seems to be much higher. Not completely sure why.
```

Comment:
maybe because the operations involved in mapping from svg to font space introduce some floating-point loss of precision?

Reply:
Plausible but not yet proven. I thought floats durable enough to survive the type of mangling we're doing but maybe that's just ... wrong.